javascript/package.json

```diff
@@ -13,7 +13,7 @@
     "prettier-standard-check": "yarn run prettier-standard --check ./**/*.js"
   },
   "dependencies": {
-    "actioncable": ">= 5.2",
+    "@rails/actioncable": ">= 6.0",
```

Comment:
Does this make it so this version relies on this package? 

Aka does this remove support for `"actioncable": ">= 5.2"`

Reply:
Yes.

Reply:
Major version bump then? @hopsoft

Reply:
Maybe. Not sure yet. The API hasn't changed enough to warrant this, but that dependency bump may warrant another major. Though I'm not really a fan of doing that on this one. 

Maybe I can switch actioncable back to the previous version. I'll try that first and see what happens.